BookingsApi/BookingsApi/Controllers/V2/HearingsControllerV2.cs

```diff
@@ -247,7 +247,45 @@ await UpdateHearingDetails(hearing.Id, requestHearing.ScheduledDateTime,
 
             return NoContent();
         }
+        
+        /// <summary>
+        /// Return hearing details for todays hearings
+        /// </summary>
+        /// <returns>Booking status</returns>
+        [HttpGet("today")]
+        [OpenApiOperation("GetHearingsForToday")]
```

Comment:
Suffix the operations with `V2` like other operations with the same name
```suggestion
        [OpenApiOperation("GetHearingsForTodayV2")]
```